src/plugins/maps_dashboards/public/index.scss

```diff
@@ -15,7 +15,3 @@
   left: 10px;
```

Comment:
Try and avoid px values. all our spacing and sizing values should ideally come from OUI/EUI spacing values: [Link](https://eui.elastic.co/v34.6.0/#/guidelines/sass:~:text=Sass%20guidelines-,Core%20variables,-Sizing)

Reply:
Yes, will update!